doc/source/mast_data_products/svm_processing.rst

```diff
@@ -674,6 +674,12 @@ gets used to define the value of this keyword, and can be called directly for an
     `Harris corner detection algorithm <https://scikit-image.org/docs/0.7.0/auto_examples/plot_harris.html>`_
     used to identify the corners.
 
+Skycell Information
+^^^^^^^^^^^^^^^^^^^
+The pipeline (ipppssoot) and SVM processing both add the SKYCELL keyword to the headers of the input FLT(C) and drizzled DRZ(C) products. This keyword includes all of the skycells that the input exposures overlap, and is used as one of the inputs for MVM processing.
```

Comment:
I am not sure what you mean by the phrase _and is used as one of the inputs for MVM processing_.  To invoke MVM processing all one really needs are the input filenames.  All the remaining information can be computed on-the-fly just as can be done for SVM.

Reply:
You're totally right, I forgot that was the case. My personal functions that I use require the _skycell_ and I forgot that the actual call to hapmultisequencer doesn't need it. Thanks for catching that.

Reply:
comment updated